metrics/cli/docs/05_user_manual_steps.md

```diff
@@ -0,0 +1,163 @@
+https://github.com/platform-platform/monorepo/blob/master/docs/09_firebase_deployment.md#google-sign-in-allowed-domains-configuration
+
+Consider the following link for a detailed structure of the "feature_config" collection:
+https://github.com/platform-platform/monorepo/blob/master/docs/19_security_audit_document.md#the-feature_config-collection
+
+Press any key to continue when you finish with the Firestore database initialization:
+```
+
+### Firebase Analytics configuration
+
+To make the Metrics Web application send the analytics data to the Firestore, the user should enable the Firestore Analytics service in the Firestore console. 
+
+During the deployment process, we should display the following prompt explaining how to enable the Firestore analytics:
+
+```text
+To enable Firebase analytics for the currently deploying project, follow the next steps:
+
+1. Follow the link and click an "Enable Google Analytics" button: https://console.firebase.google.com/project/project_id/settings/integrations/analytics
+
+Once you are done, press any key to continue:
+```
+
+### Firebase auth configuration
+
+To simplify the Firebase auth configuration, we are going to provide instructions on how to configure it for the Metrics application during the deployment process. 
+
+These instructions will contain information on how to configure the following auth providers: 
+- [Google](https://github.com/platform-platform/monorepo/blob/master/docs/09_firebase_deployment.md#firebase-google-sign-in-configuration)
+- [Email and Password](https://github.com/platform-platform/monorepo/blob/master/docs/09_firebase_deployment.md#firebase-email-and-password-sign-in-configuration)
+
+Consider the following prompt instructions helping to configure the Firebase Auth:
+
+```text
+To enable the Firebase Auth for the Metrics Web application, consider the following steps:
+
+1. Follow the link and click a "Get started" button: https://console.firebase.google.com/project/project_id/authentication
+2. Navigate to the 'Sign-in method' tab on the Firebase auth page or use the following link: https://console.firebase.google.com/project/project_id/authentication/providers
+3. Enable an Email/Password provider by clicking on provider name and toggling the `Enable` switch.
+4. Enable a Google provider by clicking on provider name and toggling the `Enable` switch.
+5. On the 'Google' provider popup, open the 'Web SDK configuration' tab, copy the Web client ID and paste it to the console
+
+Paste your Web client Id here:
+```
+
+### Sentry Configuration
+
+Since the Metrics Web application uses Sentry, we should provide detailed instruction to help a user configure a Sentry release during the Metrics application deployment process.
+Let's review prompts instructions that will help configure a new Sentry release.
+
+The first prompt for authorizing to the Sentry and retrieving an `Organization Slug`:
```

Comment:
Would it be great to add a bit more info on what is `Organization Slug`, `Project Slug`, and `DSN` and why do we need them?